elasticsearch-buildpacks.html.md.erb

```diff
@@ -81,7 +86,8 @@ my-elasticsearch   elasticsearch   xxsmall   my-kibana, my-logstash   create suc
 ```
 
 ### <a id='cf-cups'></a> Create a User Provided Service
-`cf cups my-logstash-drain -l https://URL-OF-LOGSTASH-INSTANCE`
+`cf cups my-logstash-drain -l https://USERNAME:PASSWORD@URL-OF-LOGSTASH-INSTANCE`
+USERNAME & PASSWORD: This is the username and passwort you configured in the `Logstash` file.
```

Comment:
+USERNAME & PASSWORD: the username and the password you configured in the `Logstash` file